firmware/application/apps/ui_debug.hpp

```diff
@@ -263,6 +264,39 @@ class DebugControlsView : public View {
         "Done"};
 };
 
+class DebugPmemView : public View {
+   public:
+    DebugPmemView(NavigationView& nav);
+    void focus() override;
+    bool on_encoder(const EncoderEvent delta) override;
+    std::string title() const override { return "Pmem"; }
+    int32_t page{0};
+    static constexpr uint8_t page_size{96};  // Must be multiply of 4 otherwise bit shifting for register view wont work properly
+    static constexpr uint8_t page_max{(portapack::memory::map::backup_ram.size() + page_size - 1) / page_size - 1};
```

Comment:
Something seems off about this math. I don't thing you need the last `- 1`.
This is basically finding an alignment and that math is usually `x + (n-1) / n`

Reply:
Well I was in the same mental trap and found myself watching a page full of FFs 🤣 
This actually doesn't calculate the number of pages but the id of the last page, to feed into the "max(0, min(page_max", and avoid unnecessary subtraction and/or a custom "min" there..